src/renderer/data.ts

```diff
@@ -90,8 +90,10 @@ export const defaultConfiguration: Configuration = {
                         {
                             name: 'Experimental',
                             key: 'experimental',
-                            url: 'https://cdn.flybywiresim.com/addons/a32nx/experimental',
-                            alternativeUrls: [],
+                            url: 'https://github.com/flybywiresim/a32nx/releases/download/assets/experimental/',
+                            alternativeUrls: [
+                                'https://cdn.flybywiresim.com/addons/a32nx/experimental'
```

Comment:
Should have a trailing comma? At least the standard FBW eslint config requires that...

Reply:
The linting config here doesn't, but I can add it if you want to

Reply:
Never mind. It doesn't use https://github.com/flybywiresim/eslint-config-flybywire

Reply:
#381